.gitattributes

```diff
@@ -1,3 +1,5 @@
 src/doc		export-ignore
 .gitattributes	export-ignore
 .gitignore	export-ignore
+
+*.C linguist-language=C++
```

Comment:
GitHub seemed to be syntax highlighting ESMF .C source files as C rather than C++. This seems to fix it.